_posts/2021-11-18-increasing-pulpcore-compatibility-with-plugin-versions.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Increasing Pulpcore Compatibility with Plugin Versions
+author: Brian Bouterse
+---
+
+## Tightly coupled plugins and pulpcore
+
+One challenge users and plugin maintainers face is a close coupling between plugin releases and the
+corresponding pulpcore versions. For example, pulp_rpm 3.15.z supports pulpcore 3.15 and 3.16, but
+not 3.17. Users wanting to upgrade to pulpcore 3.17 need to upgrade pulp_rpm too which makes
+upgrades harder. For plugin maintainers, e.g. pulp_rpm maintainers when pulpcore 3.17 is released
+they must release a pulp_rpm even if it has no meaningful changes, it has to be done in order to
+declare its safe to use with 3.17.
+
+## Solution
+At pulpcon 2021 we discussed various options and summarized them in [this discourse thread](https://discourse.pulpproject.org/t/what-to-do-to-reduce-unecessary-plugin-compatibility-releases/236/).
+The solution we will try is to have the plugin API that plugins use only include breaking plugin API
```

Comment:
`that plugins use` should be surrounded by comma?

Reply:
Maybe? It seems right without it, but I'm not a great grammarian. I'll leave unless someone tells me for sure.

Reply:
I have refrained from commenting because I haven't a good solution to this. 
I will say that I found this one sentence hard to read out of the whole blog. There's definitely something wrong with it, and I think Ina's sensing it. I haven't a clever way around it.  
You could probably drop `that plugins use` altogether.

Perhaps rewriting as: 

`As a solution, we want to try including breaking changes to the Plugin API only in specific releases.`

Reply:
Thank you @melcorr and @ipanova I'm changing it to this. This is better.